### PR TITLE
Add helper script for the build chain

### PR DESCRIPTION
**Pull request by n3f4s**
This change is [Reviewable](https://reviewable.io/reviews/dev-osrose/osirose-new/94)

### CODE REVIEW

**Comment by RavenX8** [comment]
Looks nice. Could you re-target the PR to the trunk branch?

**Review by RavenX8 — COMMENTED** [review]
I would suggest making g++-6 the default version that this requires.

**Comment by n3f4s** [comment]
I kept ``gcc-5`` because it was what was originally in the set-up script but IMHO ``gcc`` and ``g++`` should be used (without any version or path specification). As long as the compiler support ``c++14`` (and the other flags), the version won't cause problem (compiler bug aside). Specifying the path can be annoying for people that use link to modify the behaviour of the compiler (for example using ``/usr/local/bin/gcc`` as a link to ``ccache``).

There is some bug in the scripts that I have corrected but I have not pushed the fix yet (I will do it as soon as I can).

**Comment by RavenX8** [comment]
I agree with the gcc and g++ should be used instead of the gcc-<version> and g++-<version>. The next time I go through and fix up the CMake files I was going to convert them. The only reason why it currently uses the path is due to a issue on the Travis-CI where we can't overwrite set the default compiler.

File: scripts/default-env.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export PROOT=/home/vincent/dev/osIROSE-new
```
Comment (L3nn0x) [inline scripts/default-env.sh]: You shouldn't hard code the paths. We don't all have a folder named vincent ;)

Comment (n3f4s) [reply scripts/default-env.sh]: oups, my bad, should have checked that ... (It's not my fault if bash can't read my mind and not expand the variable when it should but that's not what I wanted to do ! ^^  )

**Review by RavenX8 — APPROVED** [review]

**Review by L3nn0x — APPROVED** [review]

**Pull request merged**